templates/Template.NEP5.CSharp/NEP5.cs

```diff
@@ -0,0 +1,85 @@
+using Neo.SmartContract.Framework;
+using Neo.SmartContract.Framework.Services.Neo;
+using System;
+using System.ComponentModel;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+namespace Template.NEP5.CSharp
+{
+    [Features(ContractFeatures.HasStorage | ContractFeatures.Payable)]
+    public partial class NEP5 : SmartContract
+    {
+        #region Token Settings
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static string Name() => "Token Name";
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static string Symbol() => "TokenSymbol";
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static byte Decimals() => 8;
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static ulong MaxSupply() => 1_000_000_000;
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static ulong InitialSupply() => 20_000_000;
```

Comment:
```suggestion
        [MethodImpl(MethodImplOptions.AggressiveInlining)]
        public static byte Decimals() => 8;
        [MethodImpl(MethodImplOptions.AggressiveInlining)]
        public static ulong MaxSupply() => 10_000_000_000_000_000;
        [MethodImpl(MethodImplOptions.AggressiveInlining)]
        public static ulong InitialSupply() => 2_000_000_000_000_000;
```
Since the decimals is 8 here, supply number should be adjusted else it will cause issues when invoke `mint`.